core/src/main/scala/org/bitcoins/core/psbt/PSBT.scala

```diff
@@ -852,7 +847,7 @@ object PSBT extends Factory[PSBT] {
 
     def map[T](func: (
         ScriptSignatureParams[InputInfo],
-        Option[BaseTransaction]) => T): Vector[T] = {
+        Option[Transaction]) => T): Vector[T] = {
```

Comment:
ditto

Reply:
because now witness txs need the previous tx